src/app/lib/utilities/ichefURL/index.js

```diff
@@ -18,9 +18,27 @@ const buildPlaceholderSrc = (src, resolution) => {
   return newUrl.join('/');
 };
 
+const buildPipsPlaceholderSrc = (src, resolution) => {
+  const parts = src.split('/');
+  const [protocol, blank, domain, type, imgService, ...extraParts] = parts;
```

Comment:
are you sure pips images all follow this format?

Reply:
I am basing from previous tests implemenation I am assuming yes. The format is as follows for all the links ichef.test.bbci.co.uk/images/ic/$widthxn/p01kdbpk.jpg, and https://ichef.test.bbci.co.uk/images/ic/$widthxn/p01kdbpk.jpg.